CMakeLists.txt

```diff
@@ -104,12 +104,20 @@ option(REALM_ENABLE_ASSERTIONS "Enable assertions in release mode." OFF)
 option(REALM_ENABLE_ALLOC_SET_ZERO "Zero all allocations." OFF)
 option(REALM_ENABLE_ENCRYPTION "Enable encryption." ON)
 option(REALM_ENABLE_MEMDEBUG "Add additional memory checks" OFF)
+option(REALM_SANITIZE_ADDRESS "Enable the GCC/Clang address sanitizer." OFF)
```

Comment:
No need for this. There is already a toolchain for specialty builds: https://github.com/realm/realm-core/blob/master/tools/cmake/SpecialtyBuilds.cmake